platform/src/core/urls/airqlouds.js

```diff
@@ -0,0 +1,10 @@
+import { stripTrailingSlash } from '../utils/strings';
+
+const BASE_DEVICE_REGISTRY_URL = stripTrailingSlash(
+  process.env.NEXT_PUBLIC_BASE_AIRQLOUDS_URL ||
+    process.env.NEXT_PUBLIC_API_BASE_URL,
```

Comment:
Can we update the [.env.sample](https://github.com/airqo-platform/AirQo-frontend/blob/ft-plat-airqlouds-toggle/platform/.env.example) file to include these variables, with the respective values for none sensitive information and dummy values for sensitive information?